.storybook/common.ts

```diff
@@ -186,3 +187,7 @@ export interface StorybookCustomEvent {
   name: string;
   payload: any;
 }
+
+export function addStorybookListeners(events) {
+  events.forEach(e => addEventListener(e, logEvent));
```

Comment:
I don't think the linter runs in this file, but we should be consistent anyway.

```suggestion
  events.forEach((e) => addEventListener(e, logEvent));
```